usaspending_api/broker/management/commands/update_transactions.py

```
@@ -0,0 +1,525 @@
+
+                transaction_contract = TransactionFPDS(transaction=transaction, **contract_instance)
+                # catch exception and do nothing if we see
+                # "django.db.utils.IntegrityError: duplicate key value violates unique constraint"
+                try:
+                    transaction_contract.save()
+                except IntegrityError:
+                    pass
+
+    def add_arguments(self, parser):
+        
+        parser.add_argument(
+            '--fiscal_year',
+            dest="fiscal_year",
+            nargs='+',
+            type=int,
+            help="Year for which to run the historical load"
+        )
+        
+        parser.add_argument(
+            '--assistance',
+            action='store_true',
+            dest='assistance',
+            default=False,
+            help='Runs the historical loader only for Award Financial Assistance (Assistance) data'
+        )
+        
+        parser.add_argument(
+            '--contracts',
+            action='store_true',
+            dest='contracts',
+            default=False,
+            help='Runs the historical loader only for Award Procurement (Contract) data'
+        )
+
+        parser.add_argument(
+            '--page',
+            dest="page",
+            nargs='+',
+            type=int,
+            help="Page for batching and parallelization"
+        )
+
+        parser.add_argument(
+            '--limit',
+            dest="limit",
+            nargs='+',
+            type=int,
+            help="Limit for batching and parallelization"
+        )
+
+    # @transaction.atomic
+    def handle(self, *args, **options):
+        logger.info('Starting historical data load...')
+
+        db_cursor = connections['data_broker'].cursor()
+        fiscal_year = options.get('fiscal_year')
+        page = options.get('page')
+        limit = options.get('limit')
+
+        if fiscal_year:
+            fiscal_year = fiscal_year[0]
```

same question as above about silently discarding >1 argument